setup.py

```diff
@@ -1,9 +1,14 @@
 from setuptools import setup, find_packages
 
 
+# Do not import any validator modules here, as that would require a correct
+# dependency environment that setup.py is not guaranteed to have.
+execfile("validator/_version.py")
+
+
```

Comment:
That was my initial plan. The main problem is that if we do that, we need to make sure `validator` can always be imported without installing any dependencies, since `setup.py` is expected to run without them. And if one of us breaks that, we're not likely to notice, since we'll probably have the dependencies installed anyway. I spent some time looking for alternative ways to single source the version, and everything I read also discouraged against importing `__init__` from `setup.py`. This was the most reasonable alternative I found.